cypress-tests/cypress/support/pageBuilder/createPageTemplate.ts

```diff
@@ -4,7 +4,21 @@ declare global {
     // eslint-disable-next-line @typescript-eslint/no-namespace
     namespace Cypress {
         interface Chainable {
-            createPageTemplate(data: any): Promise<any>; // Update the data type as needed
+            createPageTemplate(data: {
+                title: string;
+                slug: string;
+                tags: string[];
+                description: string;
+                layout: string;
+                pageCategory: string;
+            }): Promise<{
+                title: string;
+                slug: string;
+                tags: string[];
+                description: string;
+                layout: string;
+                pageCategory: string;
```

Comment:
This is good, but it's not what the function returns in L53.

Change L53 then to:

```
.then(response => response.pageBuilder.pageTemplate);
```